themis/languages/java/installation-android.md

```diff
@@ -5,7 +5,7 @@ title:  Installation for Android
 
 # Installing JavaThemis for Android development
 
-JavaThemis for Android is [available on _JFrog Bintray_ via **Maven**](https://bintray.com/cossacklabs/maven/themis).
+JavaThemis for Android is [available on JFrog Bintray via **JCenter** and **Maven**](https://bintray.com/cossacklabs/maven/themis).
```

Comment:
Maven is a tool. Bintray and JCenter are repositories.
```suggestion
JavaThemis for Android is [available on JCenter and Bintray via **Maven**](https://bintray.com/cossacklabs/maven/themis).
```

Reply:
thank you, I'm being constantly confused by this